code/modules/admin/verbs/asays.dm

```diff
@@ -0,0 +1,63 @@
+GLOBAL_LIST_EMPTY(asays)
+
+/datum/asays
+	var/ckey = ""
+	var/rank = ""
+	var/message = ""
+	var/time = 0
+
+/client/proc/view_asays()
+	set name = "View Asays"
```

Comment:
I'd change this to "Asays" only or such. Admins will look for asays and thus look at the A first. Atleast that's what I heard from some of them.

Reply:
Good point

Reply:
Still applies

Reply:
Whoops, fixed